### PR TITLE
logging: Simplify imports and initialization

### LINKED ISSUE #2254: Clean up common logging code
**Information**
latest master branch

**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
The logging code used in every module seems a bit uncommon and worsens readability in my opinion. Therefore, I suggest a small clean up.

The existing code relies on:
```
from logging import INFO as _INFO
from logging import getLogger
...
_log = getLogger(__name__)
del getLogger

if _log.isEnabledFor(_INFO):
    _log.info("Log message")
```

It is not good practice and haven't seen this practice before and I would also stick to the common names, which every Python developers knows.

```
import logging
...
logger = getLogger(__name__)

if logger.isEnabledFor(logging.Info):
    logger.info("Log message")
```

### PR DESCRIPTION
**Pull request by MattHag**
Fixes #2254

### CODE REVIEW

**Comment by pfps** [comment]
I get errors in this PR

```
idefix Solaar> bin/solaar -d
2024-02-10 09:38:06,812,812    ERROR [MainThread] logitech_receiver.diversion: failed to load from /home/pfps/.config/solaar/rules.yaml
'>=' not supported between instances of 'int' and 'function'
2024-02-10 09:38:07,008,008  WARNING [MainThread] hidapi.udev: Report Descriptor not processed for DEVICE /dev/hidraw6 BID 001F VID 00008087 PID 00000AC2: unpack requires a buffer of 2 bytes
2024-02-10 09:38:07,008,008  WARNING [MainThread] hidapi.udev: Report Descriptor not processed for DEVICE /dev/hidraw7 BID 001F VID 00008087 PID 00000AC2: unpack requires a buffer of 2 bytes
2024-02-10 09:38:07,021,021  WARNING [MainThread] hidapi.udev: Report Descriptor not processed for DEVICE /dev/hidraw4 BID 0018 VID 0000056A PID 000051E3: Unsupported global tag: 0b101
2024-02-10 09:38:07,022,022  WARNING [MainThread] hidapi.udev: Report Descriptor not processed for DEVICE /dev/hidraw2 BID 0018 VID 000006CB PID 0000CD3E: Unsupported global tag: 0b101
2024-02-10 09:38:07,129,129    ERROR [ReceiverListener:hidraw5] logitech_receiver.listener: processing Notification(10,1,41,0C,327F40)
Traceback (most recent call last):
  File "/usr/lib64/python3.12/logging/__init__.py", line 1798, in isEnabledFor
    return self._cache[level]
           ~~~~~~~~~~~^^^^^^^
KeyError: <function debug at 0x7f6eadd2c040>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/listener.py", line 176, in run
    self._notifications_callback(n)
  File "/home/local/SoftwareDownloads/Solaar/lib/solaar/listener.py", line 276, in _notifications_handler
    _notifications.process(dev, n)
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/notifications.py", line 58, in process
    return _process_device_notification(device, status, notification)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/notifications.py", line 176, in _process_device_notification
    return _process_hidpp10_notification(device, status, n)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/notifications.py", line 286, in _process_hidpp10_notification
    status.changed(active=link_established)
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/status.py", line 281, in changed
    self.read_battery(timestamp)
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/status.py", line 250, in read_battery
    battery = self._device.battery()
              ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/device.py", line 400, in battery
    result = _hidpp20.get_battery(self, battery_feature)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/hidpp20.py", line 1852, in get_battery
    result = battery_function(device)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/hidpp20.py", line 1776, in get_battery_voltage
    return decipher_battery_voltage(report)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/local/SoftwareDownloads/Solaar/lib/logitech_receiver/hidpp20.py", line 1804, in decipher_battery_voltage
    if logger.isEnabledFor(logging.debug):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/logging/__init__.py", line 1802, in isEnabledFor
    if self.manager.disable >= level:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: '>=' not supported between instances of 'int' and 'function'
```

**Comment by pfps** [comment]
@MattHag See previous comment about errors.

**Comment by MattHag** [comment]
I see my mistake

File: lib/solaar/gtk.py [hunk 1]
```diff
@@ -117,7 +114,7 @@ def _parse_arguments():
         logging.getLogger('').addHandler(stream_handler)
 
     if not args.action:
-        if _log.isEnabledFor(logging.INFO):
+        if logger.isEnabledFor(logging.INFO):
             logging.info('version %s, language %s (%s)', __version__, _i18n.language, _i18n.encoding)
```
Comment (MattHag) [inline lib/solaar/gtk.py:118]: Shouldn't this be logger.info as well?

**Comment by pfps** [comment]
@MattHag Yes.

**Review by pfps — APPROVED** [review]
Looks good now

**Pull request merged**